src/components/MenuContent/MenuContent.css

```diff
@@ -1,14 +1,20 @@
 @import '../../marketplace.css';
 
 .root {
-  position: absolute;
   z-index: var(--zIndexPopup);
 
   background-color: var(--matterColorLight);
   border-top: 1px solid var(--matterColorNegative);
   box-shadow: var(--boxShadowPopup);
   border-radius: 0 0 2px 2px;
   transition: var(--transitionStyleButton);
+
+  /* On mobile, the popup will take full screen width. */
+  position: fixed;
```

Comment:
Does this mean that the filter will move along when user is scrolling and the parent component has enough height for some reason?

How about using `width: 100vw;` instead?

Reply:
or calc(100vw - 48px)

Reply:
Hmm, the top seems to stay where the parent was when the dropdown is rendered. Even if the page is scrolled and the parent moves. In search page this is ~ok since the parent becomes sticky and doesn't move that much.

The full width itself isn't enough, the coordinates for absolute position is in relation to the parent, not the screen. So calculating the horizontal position from the parent is a lot more complex.

Reply:
This is now fixed, see the latest commits.